Content.Client/Follower/FollowerUpdateSystem.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Content.Shared.Follower.Components;
+
+namespace Content.Client.Follower;
+
+/// <summary>
+/// This handles updating the position of every following entity, as it no longer uses parenting.
+/// </summary>
+public sealed class FollowerUpdateSystem : EntitySystem
```

Comment:
You _may_ wish to make this explicitly run before physics does, so to prevent random chance of one frame latency. @metalgearsloth